.jenkins/scripts/global/init-docker.sh

```diff
@@ -0,0 +1,15 @@
+while getopts o: flag
```

Comment:
In my opinion, while this is fine, it is an awkward bash script to have. It does not fully encapsulate the docker setup (e.g. missing docker install) and complicates a series of simple commands with the need for passing args and parsing params.